src/Core/Rexl.Bind/Strings/RexlStrings.cs

```diff
@@ -3,6 +3,8 @@
 
 // WARNING: This .cs file is generated from the corresponding .tt file. DO NOT edit this .cs directly.
 
+using System;
```

Comment:
How did this get added?

Reply:
I'm not sure, I've removed it now though